Android/JatinGoyal/Beginner/SplashScreen/app/src/main/java/com/example/splashscreen/SplashActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.example.splashscreen3;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+public class SplashActivity extends Activity {
```

Comment:
This activity is not required. The ViewPager is supposed to be the splash screen